rest-api/models/pokemon_handler.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import (
+	"database/sql"
+	"strconv"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func SavePokemon(id int, name string) {
+	db, err := sql.Open("mysql", "poke:pokepassword@tcp(127.0.0.1:3306)/pokedex")
```

Comment:
Just as a recommendation try to use a ".env" file to hide sensitive information like this db credentials

Reply:
this one has been added, thanks for the feedback , I'm using viper to set a yaml file in there